src/gcp_scanner/test_unit.py

```diff
@@ -208,6 +208,30 @@ def test_get_scope_from_rt_no_scope(self, mocked_post):
     )
 
 
+class TestScopesIntegration(unittest.TestCase):
+  """Integration test against live test-porject."""
+  def setUp(self):
+    # TODO: get_creds_from_metadata or some other method should
```

Comment:
May be open an issue describing what need to be done and reference it here?

Reply:
This way we can have more context.

Reply:
I have added this issue #69 for this, should we create another one ?

Reply:
Ah, I see my bad. Could you please reference the issue in the code? e.g. i#69

Reply:
sure